apps/web/next.config.js

```diff
@@ -438,6 +438,22 @@ const nextConfig = {
         // COEP require-corp header is set conditionally when flag.coep is set to true
         headers: [CORP_CROSS_ORIGIN_HEADER],
       },
+      {
+        source: "/:path*",
+        has: [
+          {
+            type: "host",
+            value: "cal.com",
+          },
+        ],
+        headers: [
+          // make sure to pass full referer URL for booking pages
+          {
+            key: "Referrer-Policy",
+            value: "no-referrer-when-downgrade",
+          },
+        ],
+      },
       // These resources loads through embed as well, so they need to have CORP_CROSS_ORIGIN_HEADER
```

Comment:
what are these for?

Reply:
This is to update the `Referrer-Policy` to [`no-referrer-when-downgrade`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLLinkElement/referrerPolicy#no-referrer-when-downgrade) to pass the full referrer URL for conversion tracking ([reference](https://dub.co/help/article/tracking-internal-traffic#how-to-enable-full-referer-url-tracking-via-referer-policy-header))